packages/theme/src/color-ramps/lib/index.ts

```diff
@@ -69,9 +69,9 @@ function calculateRamp( {
 		keyof Ramp,
 		{ color: string; warning: boolean }
 	>;
-	let SATISFIED_ALL_CONTRAST_REQUIREMENTS = true;
-	let UNSATISFIED_DIRECTION: RampDirection = 'lighter';
-	let MAX_WEIGHTED_DEFICIT = 0;
+	let MAX_DEFICIT = -Infinity;
+	let MAX_DEFICIT_DIRECTION: RampDirection = 'lighter';
+	let MAX_DEFICIT_STEP = 'none';
```

Comment:
Minor, but maybe while we're already here: These are _not_ constants (both in the fact that they're function specific and we reassign their value), so I don't think they should be capitalized like this.

Reply:
I agree, changed the names to normal `maxDeficit`-like ones.